src/components/Common/SearchBar .jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Col, Row } from 'react-bootstrap';
+import { ExportToExcel } from '../Utilities/Table/ExportToExcel';
+
+const SearchBar = ({
```

Comment:
Have a relative name , So that when new developer comes he easily understand what the file is about.

Reply:
resolved